caiman/summary_images.py

```diff
@@ -694,20 +694,30 @@ def local_correlations_movie(file_name,
 
 def local_correlations_movie_offline(file_name,
```

Comment:
It'll be nice if this could work with a list of files. Probably it can since `cm.load` can load directly a list of files.